packages/server/tests/acceptance/rateLimiter.spec.ts

```diff
@@ -155,6 +158,46 @@ describe('@ratelimiter Rate Limiters Acceptance Tests', function () {
 
         await expect(relay.call(testConstants.ETH_ENDPOINTS.ETH_SEND_RAW_TRANSACTION, [signedTx], requestId)).to.be
           .fulfilled;
+        const remainingHbarsAfter = Number(await metrics.get(testConstants.METRICS.REMAINING_HBAR_LIMIT));
+        expect(remainingHbarsAfter).to.be.eq(remainingHbarsBefore);
+      });
+
+      it('should deploy a large contract and decrease remaining HBAR in limiter when transaction data is large', async function () {
+        const remainingHbarsBefore = Number(await metrics.get(testConstants.METRICS.REMAINING_HBAR_LIMIT));
+        expect(remainingHbarsBefore).to.be.gt(0);
+
+        const largeContract = await Utils.deployContract(
+          largeContractJson.abi,
+          largeContractJson.bytecode,
+          accounts[0].wallet,
+        );
+        await largeContract.waitForDeployment();
+        const remainingHbarsAfter = Number(await metrics.get(testConstants.METRICS.REMAINING_HBAR_LIMIT));
+        expect(largeContract.target).to.not.be.null;
+        expect(remainingHbarsAfter).to.be.lt(remainingHbarsBefore);
```

Comment:
Over time we can improve this beyond a `lt` but for now this is great capturing the metrics usage